templates/resources.html

```diff
@@ -157,7 +157,8 @@ <h3 class="p-heading--4">
       </div>
       <div class="col-4 u-align--center">
         <img
-          src="https://assets.ubuntu.com/v1/196ebe18-MAAS+Kubernetes.svg"
+          src="https://assets.ubuntu.com/v1/a9954c8e-hp-moonshot-server.jpg"
+          alt="MAAS and Kubernetes are a great combination for bare metal K8s."
           width="200"
           height="200"
```

Comment:
This width and height are no longer correct

Reply:
please advise how they should be set. should it use same aspect ratio? i dont really know what values to use here.

Reply:
Sure the width is 300 and the height is 237